prime-angular-frontend/src/app/modules/health-auth-site-reg/pages/health-auth-care-type-page/health-auth-care-type-page.component.spec.ts

```diff
@@ -16,6 +17,18 @@ import { HealthAuthCareTypePageComponent } from './health-auth-care-type-page.co
 import { HealthAuthSiteRegRoutes } from '@health-auth/health-auth-site-reg.routes';
 
 describe('HealthAuthCareTypePageComponent', () => {
+  const mockActivatedRoute = {
+    snapshot: {
+      data: {
+        title: 'Technical Support',
```

Comment:
Is there a reason this is 'Technical Support'?  'Health Authority Care Type' might be more accurate